release_bot/configuration.py

```diff
@@ -43,9 +45,29 @@ def __init__(self):
         self.github_app_installation_id = ''
         self.github_app_id = ''
         self.github_app_cert_path = ''
+
         self.clone_url = ''
         self.webhook_handler = False
 
+        # when you need to have different name for pypi release
+        self.pypi_project = ''
+
+
+    def pypi_project_from_setup_cfg():
+
+        """
+        Getting the name from the metadata section of setup.cfg
+        :return the pypi_project name or None
+        """
+        try:
+            pypi_config = configparser.ConfigParser()
+            pypi_config.read("setup.cfg")
+            metadata = pypi_config["metadata"]
+            return metadata["name"]
```

Comment:
I don't think there's a need for using try-except. Just use [`get`](https://stackoverflow.com/questions/11041405/why-dict-getkey-instead-of-dictkey) method for dicts.